### DISCUSSION TITLE
[Feature Request]Carrying the bedrock edition of Fence Overhaul

### DISCUSSION THREAD

**Discussion by Lonelyxiya** [ACGaming/UniversalTweaks#338] [Tweak Suggestions]
Do you have plans to bring back the BE Fence Overhaul feature?
url:https://www.curseforge.com/minecraft/mc-mods/fence-overhaul
Although it is closed source, there is a similar module that is Diagonal Fences.
url:https://www.curseforge.com/minecraft/mc-mods/diagonal-fences
It is open source
url:https://github.com/Fuzss/diagonalfences

**Comment by ghost** [comment]
meaning to integrate it
if the mod is working?

**Comment by Lonelyxiya** [comment]
> meaning to integrate it if the mod is working?
yes
